tools/actions/create-release.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -xe
+
+RELEASE_DATE=$1
+
+git node release --prepare --skipBranchDiff
+# We use it to not specify the branch name as it changes based on
+# the commit list (semver-minor/semver-patch)
+git config push.default current
+git push upstream
```

Comment:
We probably should not expect to have push permission, and instead make an API call to create the release commit using a `createCommitOnBranch` API call

Reply:
Actually, the `gh pr create` also pushes the branch so it should just works? https://github.com/cli/cli/issues/6958

Reply:
What I'm saying is: the workflow should not have push permissions, this `git push` call is problematic

Reply:
Why? Wouldn't it fall in the same circumstances as `commit-queue.yml`?

Reply:
I think we don't need that permission, so for that reason, we should not use it, it's good practice to use the smallest set of permissions as possible.

Reply:
What's your suggestion then? Sorry, I don't get it yet

Reply:
What do you think about a follow-up PR? I mean, the end goal is to create a proposal using automation, the sooner we land that the sooner we can test it and see how it behaves in practice.